src/comps/LyricsEditor.tsx

```diff
@@ -72,7 +80,7 @@ export default function LyricsEditor({ id, onSynced }) {
       ) : null}
       {error ? (
         <Message negative={true} icon>
-          <Icon name="error" />
+          <Icon name="warning" />
```

Comment:
Why?

Reply:
"error" isn't defined as a valid Icon name in the React Semantic UI typings and generated a type error, it also isn't mentioned in the docs for Semantic UI. "warning" seemed like the closest equivalent.